### ISSUE TITLE
Feature Request: Make Preload an option

### ISSUE THREAD

**Issue by e7mac** [cifkao/html-midi-player#10]
Sometimes, when there are lots of MIDI Files displayed on a page, preloading them all might be undesirable. Could it be possible to make an option `preload=False` in the `<midi-player>` ?

**Comment by cifkao** [comment]
Good idea! Maybe something like [`preload="none"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#attr-preload) as in the `<audio>` element. Possibly `preload="bare"` to load only the MIDI file and not the sound font.

There should also be a clear indication in the UI (different icon?) that the player is loading, as opposed to a disabled or error state. Which is a good idea anyway.

**Comment by e7mac** [comment]
Ideas for the loading indicator: a "shimmer" on the whole player/timeline slider, or a progress bar on the timeline slider
